doc/classes/SpatialMaterial.xml

```diff
@@ -70,86 +76,115 @@
 		</member>
 		<member name="ao_texture" type="Texture" setter="set_texture" getter="get_texture">
+			Texture that defines amount of ambient occlusion for a given point on the object.
 		</member>
 		<member name="ao_texture_channel" type="int" setter="set_ao_texture_channel" getter="get_ao_texture_channel" enum="SpatialMaterial.TextureChannel">
+			Specifies the channel of the [member ao_texture] in which the ambient occlusion information is stored. This is useful when you store the information for multiple effects in a single texture. For example if you stored metallic in the red channel, roughness in the blue, and ambient occlusion in the green you could reduce the number of textures you use.
 		</member>
 		<member name="clearcoat" type="float" setter="set_clearcoat" getter="get_clearcoat">
+			Sets the strength of the clearcoat effect. Setting to [code]0[/code] looks the same as disabling the clear coat effect.
 		</member>
 		<member name="clearcoat_enabled" type="bool" setter="set_feature" getter="get_feature" default="false">
-			If [code]true[/code], clearcoat rendering is enabled. Adds a secondary transparent pass to the material.
+			If [code]true[/code], clearcoat rendering is enabled. Adds a secondary transparent pass to the lighting calculation resulting in an added specular blob. This makes materials appear as if they have a clear coating on them.
 		</member>
 		<member name="clearcoat_gloss" type="float" setter="set_clearcoat_gloss" getter="get_clearcoat_gloss">
+			Sets the roughness of the clearcoat pass. A higher value results in a smoother clearcoat while a lower value results in a rougher clearcoat.
 		</member>
 		<member name="clearcoat_texture" type="Texture" setter="set_texture" getter="get_texture">
+			Texture that defines the strength of the clearcoat effect and the glossiness of the clearcoat. Strength is specified in the x-channel while glossiness is specified in the y-channel.
 		</member>
 		<member name="depth_deep_parallax" type="bool" setter="set_depth_deep_parallax" getter="is_depth_deep_parallax_enabled">
+			If [code]true[/code], shader will read depth texture at multiple points along the view ray to determine occlusion and parrallax. This can be very performance demanding, but results in more realistic looking depth mapping.
 		</member>
 		<member name="depth_enabled" type="bool" setter="set_feature" getter="get_feature" default="false">
 			If [code]true[/code], depth mapping is enabled (also called "parallax mapping" or "height mapping"). See also [member normal_enabled].
 		</member>
 		<member name="depth_flip_binormal" type="bool" setter="set_depth_deep_parallax_flip_binormal" getter="get_depth_deep_parallax_flip_binormal">
+			If [code]true[/code], direction of the binormal is flipped before using in depth effect. This may be necessary if you have encoded your binormals in a way that is conflicting with the depth effect.
```

Comment:
I'd add a "the", but it doesn't strike me as wrong per se as is, so it might be fine.